pkg/resources/data.go

```diff
@@ -533,6 +548,16 @@ func (d *TemplateData) KCMCloudControllersDeactivated() bool {
 	return false
 }
 
+// IsEncryptionConfigurationEnabled returns whether encryption-at-rest is configured or not.
+func (d *TemplateData) IsEncryptionConfigurationEnabled() bool {
+	return d.Cluster().Spec.Features[kubermaticv1.ClusterFeatureEncryptionAtRest] && d.Cluster().Spec.EncryptionConfiguration != nil && d.Cluster().Spec.EncryptionConfiguration.Enabled
+}
+
+// IsEncryptionActive returns whether encryption-at-rest is active on this cluster. Might return a different result than `IsEncryptionConfigurationEnabled()`.
+func (d *TemplateData) IsEncryptionActive() bool {
+	return d.Cluster().Status.HasConditionValue(kubermaticv1.ClusterConditionEncryptionInitialized, corev1.ConditionTrue)
+}
+
```

Comment:
TBH, i hate our TemplateData stuff. And these functions seems generally useful. So maybe we should, *gasp*, put them on the Cluster object directly or move them to `pkg/apis/kubermatic/v1/helper`?

Reply:
Good suggestion. Let me do that.